Codecademy/client-modules
# feat(Gamut): Add Input Stepper

## Add Input Stepper to Gamut

This moves the component introduced for weekly streaks to step through a numeric into Gamut for further use through Codecademy. The changes made to adapt this component for Gamut were:
- Allow for different sizing based on the max field (up to 999)
- Handle the case when the user clears out the field

### Testing Instructions
1. Run this branch locally
1. Go to `Atoms > Input Stepper` 
1. Verify that the stepper displays appropriately
1. Change some of the knobs for the control
1. Verify that the stepper firs the maximum values allowed up to 999

### Notes
Chrome no longer shows knobs for me, but Firefox does. If you're having difficulty seeing the knobs, try Firefox.

There are pre-existing issues visible in this change (e.g. not being able to display the code associated with the component).

---

## Merging your changes

When you are ready to merge to master and publish your changes, use the "squash and merge" button in GitHub, and follow the [PR Title Guide](https://github.com/Codecademy/client-modules#pr-title-guide) to format your PR title and write your PR merge description.

**IMPORTANT:** If your PR contains breaking changes, please remember to follow the instructions for breaking changes!
